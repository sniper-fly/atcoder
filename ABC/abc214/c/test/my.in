4
5 5 5 1
3 100 100 1
