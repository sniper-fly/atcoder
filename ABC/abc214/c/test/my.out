2
7
12
1
